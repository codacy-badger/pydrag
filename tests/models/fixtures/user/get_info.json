{
    "version": 1,
    "interactions": [
        {
            "request": {
                "method": "GET",
                "uri": "https://ws.audioscrobbler.com/2.0/?api_key=LAST_FM_API_KEY&format=json&method=user.getInfo&user=rj",
                "body": null,
                "headers": {
                    "User-Agent": [
                        "python-requests/2.20.0"
                    ],
                    "Accept-Encoding": [
                        "gzip, deflate"
                    ],
                    "Accept": [
                        "*/*"
                    ],
                    "Connection": [
                        "keep-alive"
                    ]
                }
            },
            "response": {
                "status": {
                    "code": 200,
                    "message": "OK"
                },
                "headers": {
                    "Server": [
                        "openresty/1.9.7.5"
                    ],
                    "Access-Control-Max-Age": [
                        "86400"
                    ],
                    "Connection": [
                        "keep-alive"
                    ],
                    "content-length": [
                        "747"
                    ],
                    "Date": [
                        "Sat, 10 Nov 2018 11:28:04 GMT"
                    ],
                    "Access-Control-Allow-Origin": [
                        "*"
                    ],
                    "Content-Type": [
                        "application/json"
                    ],
                    "Access-Control-Allow-Methods": [
                        "POST, GET, OPTIONS"
                    ]
                },
                "body": {
                    "string": "{\"user\":{\"playlists\":\"0\",\"playcount\":128549,\"gender\":\"n\",\"name\":\"RJ\",\"subscriber\":\"1\",\"url\":\"https:\\/\\/www.last.fm\\/user\\/RJ\",\"country\":\"United Kingdom\",\"image\":[{\"size\":\"small\",\"#text\":\"https:\\/\\/lastfm-img2.akamaized.net\\/i\\/u\\/34s\\/CENSORED.png\"},{\"size\":\"medium\",\"#text\":\"https:\\/\\/lastfm-img2.akamaized.net\\/i\\/u\\/64s\\/CENSORED.png\"},{\"size\":\"large\",\"#text\":\"https:\\/\\/lastfm-img2.akamaized.net\\/i\\/u\\/174s\\/CENSORED.png\"},{\"size\":\"extralarge\",\"#text\":\"https:\\/\\/lastfm-img2.akamaized.net\\/i\\/u\\/300x300\\/CENSORED.png\"}],\"registered\":{\"unixtime\":1037793040,\"#text\":1037793040},\"type\":\"alum\",\"age\":\"0\",\"bootstrap\":\"0\",\"realname\":\"Richard Jones \"}}"
                }
            }
        }
    ]
}
