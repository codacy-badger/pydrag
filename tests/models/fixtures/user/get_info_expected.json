{
    "playlists": 0,
    "playcount": 128549,
    "gender": "n",
    "name": "RJ",
    "url": "https://www.last.fm/user/RJ",
    "country": "United Kingdom",
    "image": [
        {
            "size": "small",
            "text": "https://lastfm-img2.akamaized.net/i/u/34s/CENSORED.png"
        },
        {
            "size": "medium",
            "text": "https://lastfm-img2.akamaized.net/i/u/64s/CENSORED.png"
        },
        {
            "size": "large",
            "text": "https://lastfm-img2.akamaized.net/i/u/174s/CENSORED.png"
        },
        {
            "size": "extralarge",
            "text": "https://lastfm-img2.akamaized.net/i/u/300x300/CENSORED.png"
        }
    ],
    "age": 0,
    "registered": {
        "timestamp": 1037793040,
        "text": "1037793040"
    },
    "real_name": "Richard Jones "
}
